app/frontend/src/components/designer/FormViewerMultiUpload.vue

```diff
@@ -311,6 +312,65 @@ export default {
         this.progress = true;
         this.$emit('toggleBlock', true);
         const formHtml = document.getElementById('validateForm');
+        //Add custom validation to the Components those are not covered by FormIO Validation class
+        await Utils.eachComponent(
+          this.formSchema.components,
+          function (component) {
+            //Need to cover Validation parts those are not performed by FormIO validate funciton
+            switch (component.type) {
+              case 'number':
+                component.validate.custom =
+                  "if(component.validate.required === true){valid = _.isNumber(input) ? true : 'Only number allowed on a number field';} else if(input){valid = _.isNumber(input) ? true : 'Only number allowed on a number field.';}" +
+                  component.validate.custom;
+                break;
+
+              case 'simplenumber':
+                component.validate.custom =
+                  "if(component.validate.required === true){valid = _.isNumber(input) ? true : 'Only number allowed on a simple number field';} else if(input){valid = _.isNumber(input) ? true : 'Only number allowed on a simple number field';}" +
+                  component.validate.custom;
+                break;
+
+              case 'simplenumberadvanced':
+                component.validate.custom =
+                  "if(component.validate.required === true){valid = _.isNumber(input) ? true : 'Only number allowed on a simple number advanced field';} else if(input){valid = _.isNumber(input) ? true : 'Only number allowed on a simple number advanced field';}" +
+                  component.validate.custom;
+                break;
+
+              case 'simpledatetimeadvanced':
+                component.validate.custom =
+                  "if(component.validate.required === true){valid = moment(input, _.replace('" +
+                  component.widget.format +
+                  "','dd','DD'), true).isValid() === true ? true : 'Wrong DateTime format.';} else if(input){valid = moment(input, _.replace('" +
+                  component.widget.format +
+                  "','dd','DD'), true).isValid() === true ? true : 'Wrong DateTime format.';}" +
```

Comment:
This should have an example format for a valid DateTime.

Reply:
At this point in the code when we generate this report we don't have access to the format selected by the form Designer. So adding a static date time with a particular format can mislead the form submitter